spec/truffle/ffi/pointer_spec.rb

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 2.0, or
+# GNU General Public License version 2, or
+# GNU Lesser General Public License version 2.1.
+
+require 'ffi'
+
+describe "FFI#Pointer" do
+  describe "#inspect" do
+    it "should include the address" do
+      FFI::Pointer.new(1234).inspect.should =~ /address=0x0*4d2/
+    end
+
+    it "should not include the size if the pointer is unsized" do
+      FFI::Pointer.new(1234).inspect.should_not =~ /size=/
+    end
+
+    it "should include the size if there is one" do
+      FFI::MemoryPointer.new(:char, 16).inspect.should =~ /size=16/
+    end
+  end
+end
```

Comment:
This test should be added to https://github.com/ffi/ffi, not as a Truffle-specific spec.
Could you make a PR there and remove the spec in this PR?
Then we'll just get the test the next time we update FFI.